src/java.base/share/classes/java/lang/ThreadGroup.java

```diff
@@ -792,6 +792,14 @@ private List<ThreadGroup> synchronizedSubgroups() {
         }
     }
 
+    /**
+     * Returns an snapshot of the subgroups as an array, used by JVMTI.
```

Comment:
The update looks good, just a typo here where it should be "a snapshot".

Reply:
fixed.